app/api/analytics/route.ts

```
@@ -0,0 +1,296 @@
+
+    // Calculate placements by partner instead of by program
+    const partnerPlacements = new Map<string, number>();
+    const acceptedApps = applications.filter((app: Application) => app.status === "OFFER_ACCEPTED");
+    
+    // Count placements by partner
+    acceptedApps.forEach((app: Application) => {
+      if (app.jobs) {
+        let partnerName = "Unknown";
+        
+        // Use partner name if available, otherwise use company name
+        if (app.jobs.partners && app.jobs.partners.name) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)